aiida/tools/importexport/common/archive.py

```diff
@@ -102,14 +198,56 @@ def filepath(self):
         """
         return self._filepath
 
+    @property
+    def output_filepath(self):
+        """Return the filepath of where the archive should be repacked
+
+        :return: the archive output filepath
+        """
+        return self._output_filepath
+
+    @property
+    def overwrite(self):
+        """Return whether repacking should be overwrite a possible existing file."""
+        return self._overwrite
+
+    @property
+    def silent(self):
```

Comment:
I don't think `silent` needs to be set or retrieved from the outside. One should just open the `Archive` class with the correct `silent` value and use it like that. Why would we need the functionality of changing this value during the context manager. These properties are not really necessary and are what contributes a lot to the bloating

Reply:
Completely agree. This is why more eyes are needed on such things :) Thanks.